src/main/frontend/components/whiteboard.cljs

```diff
@@ -292,3 +295,35 @@
     (let [name (get-in route-match [:parameters :path :name])
           {:keys [block-id]} (get-in route-match [:parameters :query])]
       (whiteboard-page name block-id))))
+
+(defn onboarding-show
+  []
+  (when (and (user-handler/feature-available? :whiteboard)
+             (not (or (state/sub :whiteboard/onboarding?)
+                      (state/enable-whiteboards?)
+                      (util/mobile?))))
+    (state/pub-event! [:whiteboard/onboarding])
+    (state/set-state! [:whiteboard/onboarding?] true)
+    (storage/set :whiteboard-onboarding? true)))
+
+(rum/defc onboarding-welcome
+  [close-fn]
+  [:div.cp__whiteboard-welcome
+   [:span.head-bg
+
+    [:strong (t :on-boarding/closed-feature (name (:whiteboard user-handler/feature-matrix)))]]
```

Comment:
It seems we only have closed alpha, but no closed beta right? I think we do not need string interpolating here.

Reply:
On a second though, having a dedicated translation for "closed beta" and "closed alpha "might be a better idea, and we would also avoid the interpretation. Is that what you meant?

Reply:
Oh, I thought beta means general availability and thus we may have "open beta" and "closed beta". However right now closed-beta means the "Backers" right? I am fine with the current solution. :D